### PR TITLE
[WP-200] Add deprecation comment

### PR DESCRIPTION
**Pull request by danielasy**
Create `Add deprecation comment` codemod to add the following message before the first ImmutableJS import:

```
// ImmutableJS usage is deprecated
// Please, do not copy & paste or use this snippet as reference :)
// How to refactor? See https://github.com/quintoandar/farewell-immutablejs/blob/master/MIGRATION.md
```

Goal: prevent developers from using outdated code as reference in new features and creating an incentive to refactor these files by linking to the guidelines.

Also :hammer::

- Link to migration guideline
- Fix How to use command
- Create Contributing section
- Fix lint
- Bump version to 1.1.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 Codemods to migrate [Immutable.js](https://github.com/immutable-js/immutable-js) to ES6.
 
+Check out the [migration guide](https://guidelines.quintoandar.com.br/#/pwa/removing-immutable) for the motivations and step-by-step instructions.
```
Comment (raphaelbs) [inline README.md:8]: Guidelines is not accessible from outside the QuintoAndar.
Maybe we should link an article on the 5A blog. There's still additional content that we could provide there like learnings and code patterns to help code refactors like this.

File: src/transforms/addDeprecationComment.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    source: {
+      value: 'immutable'
+    }
+  });
+
+  if (immutableImports.length > 0) {
+    const deprecationMessage = [
+      j.commentLine(' ImmutableJS usage is deprecated', true, false),
+      j.commentLine(' Please, do not copy & paste or use this snippet as reference :)', true, false),
+      j.commentLine(' How to refactor? See https://guidelines.quintoandar.com.br/#/pwa/removing-immutable', true, false),
```
Comment (raphaelbs) [inline src/transforms/addDeprecationComment.ts:17]: Ditto

File: README.md (hunk 1 above)
Comment (danielasy) [reply README.md]: Copied the guidelines content to this repo for now, we can add a link to the article when we publish it :slightly_smiling_face:

**Comment by danielasy** [comment]
@raphaelbs I think we should be able to run them separately, since not all projects will need the deprecation comments; maybe they are small enough that it's possible to migrate all usages in one or a few PRs. In these cases, the comments are just clutter. So, I guess it's ok for keeping them in different CLI commands

**Pull request merged**